src/Cli/Commands/ExportOptions.cs

```diff
@@ -1,27 +1,63 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Azure.DataApiBuilder.Core.Generator;
 using CommandLine;
 
 namespace Cli.Commands
 {
-    [Verb("export", isDefault: false, HelpText = "Export the GraphQL schema as a file and save to disk", Hidden = false)]
+    [Verb("export", isDefault: false, HelpText = "Export the GraphQL schema as a file and save it to disk.", Hidden = false)]
     public class ExportOptions : Options
     {
-        public ExportOptions(bool graphql, string outputDirectory, string? config, string? graphqlSchemaFile) : base(config)
+        public ExportOptions(bool graphql, string outputDirectory, string? graphqlSchemaFile,
+                bool? generate, string? samplingMode, int? numberOfRecords, string? partitionKeyPath, int? maxDays, int? groupCount, string? config) : base(config)
         {
             GraphQL = graphql;
             OutputDirectory = outputDirectory;
-            GraphQLSchemaFile = graphqlSchemaFile ?? "schema.graphql";
+            GraphQLSchemaFile = graphqlSchemaFile ?? "schema.gql";
+
+            Generate = generate ?? false;
+            SamplingMode = samplingMode ?? SamplingModes.TopNSampler.ToString();
+            NumberOfRecords = numberOfRecords;
+            PartitionKeyPath = partitionKeyPath;
+            MaxDays = maxDays;
+            GroupCount = groupCount;
         }
 
         [Option("graphql", HelpText = "Export GraphQL schema")]
         public bool GraphQL { get; }
 
-        [Option('o', "output", HelpText = "Directory to save to", Required = true)]
+        [Option('o', "output", HelpText = "Specify the directory where the schema file will be saved. This option is required.", Required = true)]
```

Comment:
change it as "specify the directory where the schema file will be saved. This option is required"